src/sentry/static/sentry/app/components/events/contexts/device/getDeviceKnownData.tsx

```diff
@@ -0,0 +1,27 @@
+import {KeyValueListData} from 'app/components/events/interfaces/keyValueList/types';
+import {getMeta} from 'app/components/events/meta/metaProxy';
+
+import getDeviceKnownDataDetails from './getDeviceKnownDataDetails';
+import {DeviceKnownDataType, DeviceData} from './types';
+
+function getOperatingSystemKnownData(data: DeviceData): Array<KeyValueListData> {
+  const knownData: Array<KeyValueListData> = [];
+
+  const dataKeys = Object.keys(data);
+  for (const key of dataKeys) {
+    const knownDataDetails = getDeviceKnownDataDetails(data, key as DeviceKnownDataType);
+
+    if (key === null || !knownDataDetails) {
```

Comment:
@priscilawebdev Just a heads up, I'm unsure if `key` can ever be `null`. I think it's implied to be a string since `Object.keys()` should be expected to return an array of strings.

Reply:
Good catch 👏 thank you for warning me!